src/modules/aws-ebs/sub-modules/config/loadEbsConfig.ts

```diff
@@ -0,0 +1,42 @@
+import { EbsConfig } from "./EbsConfig";
+
+export const isEnvironmentVariableDefined = (environmentVariable: string | undefined): boolean => {
+  return (environmentVariable && environmentVariable !== "undefined") === true;
+};
+
+export const loadEbsConfig = (): EbsConfig => {
+  let sourceAwsRegion: string;
+
+  if (process.env.AWS_REGION && process.env.AWS_REGION !== "undefined") {
+    sourceAwsRegion = process.env.AWS_REGION;
+  } else {
+    throw new Error("Env variable AWS_REGION must be defined.");
+  }
+
+  const quarantineAwsRegion: string =
+    process.env.QUARANTINE_AWS_REGION && process.env.QUARANTINE_AWS_REGION !== "undefined"
+      ? process.env.QUARANTINE_AWS_REGION
+      : process.env.AWS_REGION;
+
+  const quarantineAwsAccounts: string[] =
+    process.env.QUARANTINE_AWS_ACCOUNTS && process.env.QUARANTINE_AWS_ACCOUNTS !== "undefined"
+      ? process.env.QUARANTINE_AWS_ACCOUNTS.split(",").map(account => {
+          return account.trim();
+        })
+      : [];
```

Comment:
If `process.env.QUARANTINE_AWS_ACCOUNTS` is undefined, should we not thrown an exception? Or are we defaulting to the source account?

Reply:
Yep, it defaults to the source account.

If the `QUARANTINE_AWS_ACCOUNT` is not defined, the runbook still proceeds with copying the snapshot to the "TARGET_REGION", but skips the "sharing" of snapshot part all together.